store/store.go

```diff
@@ -1131,6 +1131,25 @@ func (s *Store) Download(name string, downloadInfo *snap.DownloadInfo, pbar prog
 		}
 	}()
 
+	if os.Getenv("SNAPPY_USE_DELTAS") == "1" && len(downloadInfo.Deltas) >= 0 {
+		downloadDir, err := ioutil.TempDir("", fmt.Sprintf("%s-deltas", name))
+		if err == nil {
+			defer os.RemoveAll(downloadDir)
+
+			_, err = s.downloadDeltas(name, downloadDir, downloadInfo, pbar, user)
+			// We revert to normal downloads if there is any error
+			if err != nil {
+				// Just log the error and continue with the normal non-delta
+				// download.
+				logger.Noticef("cannot download deltas for %s: %v", name, err)
```

Comment:
s/cannot/Cannot/, and s/succ/Succ/ below.

(lowercase on errors, uppercase on logs)